src/main/resources/stock_trades_schema.avro

```diff
@@ -0,0 +1,64 @@
+{
+  "namespace": "ksql",
+  "name": "StockTrade",
+  "doc": "Defines a hypothetical stock trade using some known simulated stock symbols.",
+  "type": "record",
+  "fields": [
+    {
+      "name": "side",
```

Comment:
Is "side" an industry term?  It's not intuitive to users, is there another term that might have more mindshare with non-fintech?

Reply:
Yes, 'side' is a very standard trading term.